runtime/common/MeasureCounts.cpp

```diff
@@ -102,7 +102,6 @@ void ExecutionResult::deserialize(std::vector<std::size_t> &data) {
       name += std::string(1, char(data[stride + i]));
 
     stride += nChars;
-    std::unordered_map<std::string, std::size_t> localCounts;
```

Comment:
The code here, `ExecutionResult::deserialize` appears to be mostly a block copy of the code below in `sample_result::deserialize`. It would be nice to combine these into a single shared implementation by way of a template function or something similar.